package/MDAnalysis/lib/nsgrid.pyx

```diff
@@ -786,27 +412,18 @@ cdef class FastNS(object):
             raise ValueError("Any of the box dimensions cannot be 0")
```

Comment:
I think that `np.allclose()` is wrong. Shouldn't it be rather something like `if np.any(box[:3] == 0.0)` (or does `np.anyclose()` exist)?